GenderPayGap.WebUI/Views/ManageOrganisations/OrganisationReportsTable.cshtml

```diff
@@ -0,0 +1,83 @@
+            <th scope="col" class="govuk-table__header">Year</th>
+            <th scope="col" class="govuk-table__header">Reporting requirement</th>
+            <th scope="col" class="govuk-table__header">Report status</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        @{
+            foreach (KeyValuePair<int, OrganisationScopeForYear> scopeForYear in Model.OrganisationScopesByYear)
+            {
+                int reportingYear = scopeForYear.Key;
+                OrganisationScopeForYear organisationScopeForYear = scopeForYear.Value;
+
+                string formattedYearText = $"{reportingYear}/{(reportingYear + 1).ToTwoDigitYear()}";
+                
+                DateTime snapshotDate = organisationScopeForYear.GetSnapshotDate();
+                DateTime deadline = snapshotDate.AddYears(1).AddDays(-1);
+                    
+                string encryptedOrganisationId = Encryption.EncryptQuerystring(Model.OrganisationId.ToString());
+                
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell govuk-!-font-weight-bold">
+                        @formattedYearText
+                    </td>
+                    
+                    <td class="govuk-table__cell">
+                        <div>
+                            @organisationScopeForYear.GetScopeVariantText()
+                            
+                            @{
+                                var reportingDeadlineText = organisationScopeForYear.GetByReportingDeadlineText(deadline);
+                            }
+                            
+                            @if (reportingDeadlineText != null)
+                            {
+                                <br/>
+                                @reportingDeadlineText
+                            }
+                        </div>
+                        @if (organisationScopeForYear.CanChangeScope()) 
+                        {
+                            <div>
+                                <a href="@(Url.Action("ChangeOrganisationScope", "Scope", new { encryptedOrganisationId = encryptedOrganisationId, reportingYear = reportingYear }))">
+                                    Change<span class="govuk-visually-hidden"> scope for year @formattedYearText</span>
+                                </a>
+                            </div>
+                        }
+                    </td>
+                    
+                    <td class="govuk-table__cell">
+                        <div>
+                            @organisationScopeForYear.GetReportStatusText()
+                        </div>
+                        <div>
+                            @if (FeatureFlagHelper.IsFeatureEnabled(FeatureFlag.NewReportingJourney))
+                            {
+                                <a class="govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-0" href="@Url.Action("ReportOverview", "ReportOverview",

```

Comment:
Github is complaining about these, which I think is inadvertently because it expects each HTML attribute to be on only one line - thoughts? i.e. a line break before the `href`

Reply:
I like the order:
```cshtml
<a href="..."
   class="govuk-link">
    Link text
</a>
```
It helps me remember that there should always be a `class="govuk-link"` on each `<a>` tag (except in this case where it's a button)

Reply:
Actually, styling links as buttons is frowned upon ☹️ for accessibility reasons (happy to have a call to discuss more details)
If these are going to be buttons, then it might be better if each one is inside a form:
```cshtml
<form method="get"
      action="@(Url.Action(...))">
    <button .../>
</form>
```
I know that's a bit of a bore, but it does genuinely help people using speech-to-command software

Reply:
Could the `aria-role` come in handy here? That way we could keep the links, style them as buttons but a screen-reader would present them as buttons?

Reply:
Ooh yes, I think that would work - great thought 😄